shiny-app/rstats-health-app/app.R

```diff
@@ -0,0 +1,55 @@
+#
+# This is a Shiny web application. You can run the application by clicking
+# the 'Run App' button above.
+#
+# Find out more about building applications with Shiny here:
+#
+#    http://shiny.rstudio.com/
+#
+
+library(shiny)
+library(shinydashboard)
+# Define UI for application that draws a histogram
+ui <-  dashboardPage(
+  dashboardHeader(title = "Rstats net Health",
+                  
+              dropdownMenu(type = "messages",
+                               messageItem(
+                                 from = "Git Version Control",
+                                 message = "[Github Status]: We have a temporary issues with webhooks.",
+                                 time = "13:50",
+                                 icon = icon("exclamation-triangle")
+                                 
+                               ),
+                               messageItem(
+                                 from = "Git Version Update",
+                                 message = "[Github Status]: Everything is operating normally.",
+                                 time = "15:30",
+                                 icon = icon("check")
+                               
+                               ),
+                               messageItem(
+                                 from = "Support",
+                                 message = "The new server is ready"
+                               )
+                               )),
+  
+  dashboardSidebar(   
+    selectInput("git-control", "Git Version Control:", 
```

Comment:
The `selectInput` only allows choosing one of the options. Could you change it instead to use `checkboxInput`? This will allow users to select the individual sources that they want included in their report. It'll look something like below:

```
checkboxInput("github", **"GitHub"),
checkboxInput("travis", "Travis CI"),
```